apps/browser/qml/pages/components/Overlay.qml

```diff
@@ -386,6 +390,30 @@ Shared.Background {
                 }
             }
 
+            OverlayListItem {
+                id: historyButton
+                height: toolBar.rowHeight
+                iconWidth: toolBar.iconWidth
+                horizontalOffset: toolBar.horizontalOffset
+
+                y: -((historyContainer.showFavorites ? -searchField.y : historyList.contentY) - height)
+                // On top of HistoryList and FavoriteGrid
+                z: 1
+
+                text: qsTrId("sailfish_browser-la-history")
+                iconSource: "image://theme/icon-m-history"
+                opacity: visible && toolBar.opacity < 0.9 ? 1.0 : 0.0
+                enabled: overlayAnimator.atTop
+                visible: historyContainer.showFavorites && !historyContainer.showHistoryList
+
+                onClicked: {
+                    var historyPage = pageStack.push("../HistoryPage.qml", { model: historyModel })
+                    historyPage.loadPage.connect(loadPage)
```

Comment:
When cancelling the history page (i.e. swipe left), the toolbar has descended, and it feels a bit strange returning to the webpage when previously the start page was open (when going back, I expect to be returned to the same place I cam from). Is it possible to have the overlay remain opening when swiping back from the history page?

Reply:
@llewelld   before the changes,  behavior was the same, is there no such behavior in the design?

Reply:
I'm don't think the design goes in to this level of detail. You're right though that other users have the same behaviour as this. It's just especially pronounced in this case because the toolbar has grown to take up so much of the screen, so it seems like a separate page. Let me double check on this.